src/platform/forms-system/src/js/review/widgets.jsx

```diff
@@ -36,11 +48,15 @@ export const yesNo = ({ value, options = {} }) => {
     displayValue = labels.N || 'No';
   }
 
-  return <span>{displayValue}</span>;
+  return (
+    <span className="dd-privacy-hidden" data-dd-action-name="data value">
```

Comment:
Do we need to hide yes/no values? If yes, then maybe an action name of "yes or no value" works better?